pyirf/scripts/make_performance.py

```diff
@@ -130,7 +130,6 @@ def main():
                 # sys.exit()
 
             psf = np.percentile(data['offset'], radius)
-            psf_err = psf / np.sqrt(len(data))
```

Comment:
This script is going to be obsolete after PR #12 (make_DL3 is basically an updated version), you can ignore it in the future